persistence/rocksdb/src/main/java/org/infinispan/persistence/rocksdb/configuration/RocksDBStoreConfiguration.java

```diff
@@ -105,6 +105,11 @@ public int expiryQueueSize() {
       return expiration.expiryQueueSize();
    }
 
+   /**
+    * @deprecated clearThreshold is no longer being used.
+    * @return the configured clear threshold
+    */
+   @Deprecated
```

Comment:
I updated my dan_ISPN-8241_RocksDBStore branch with some changes to ignore the element if the schema version is < 12 or to throw an exception otherwise.